eos/utils/kmatrix_TEST.cc

```diff
@@ -0,0 +1,264 @@
+#include <test/test.hh>
+#include <eos/utils/parameters.hh>
+#include <eos/utils/kmatrix-impl.hh>
+#include <eos/utils/complex.hh>
+#include <iostream>
+#include <memory>
+
+#include <cmath>
+
+
+using namespace test;
+using namespace eos;
+
+
+///////////////////////
+/// Simplest K matrix, 1 channel, 1 resonance
+///////////////////////
+struct PPchan11 : public KMatrix<1,1>::Channel
+{
+    PPchan11(std::string name, double m1, double m2, unsigned N_orbital, const Parameters & p) : Channel(name, m1, m2, N_orbital, {{
+        p["test::g0_1"]
+    }})
+    {
+    };
+
+    //Usefull definitions for beta and rho
+    double mm = this->_m1 - this->_m2;
+    double mp = this->_m1 + this->_m2;
+    //sqrt of the Källen factor, defined with an absolute value
+    double sqlk(const double & s) {
+        return std::sqrt(std::abs((s-mp*mp)*(s-mm*mm)));
```

Comment:
A bit of tidying up: please leave a space to the left and to the right of each binary arithmetic operator.